### ISSUE TITLE
Support for React Router 4

### ISSUE THREAD

**Issue by boldfacedesign** [blueberryapps/react-bluekit#123]
React Router 4 relies on React context to work. You can no longer simply render Link components etc. For unit testing we now have to wrap things with a <MemoryRouter> or <StaticRouter> (see https://reacttraining.com/react-router/core/guides/testing)

As those docs suggest there is possibly another workaround for this my stubbing/mocking the context.

**Comment by ondrejbartas** [comment]
Yes you can wrap whole `<MemoryRouter><BlueKit/></MemoryRouter>` and then your links starts to work 👍

**Comment by ondrejbartas** [comment]
It is same approach when you need to work with connected components in Redux ecosystem
